.github/workflows/reusable-e2e.yml

```diff
@@ -15,6 +15,25 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 16
```

Comment:
```
node-version: 16
```
Should we be using the same Node version (18) everywhere?

Reply:
ps. node 16 will be [deprecated](https://vercel.com/changelog/node-js-14-and-16-are-being-deprecated) February 16 24, so we still have time :)